pkgs/applications/video/stremio/default.nix

```
@@ -30,6 +39,8 @@ stdenv.mkDerivation rec {
     install -Dm 644 images/stremio_window.png $out/share/pixmaps/smartcode-stremio.png
     ln -s ${nodejs}/bin/node $out/opt/stremio/node
     ln -s $server $out/opt/stremio/server.js
+    wrapProgram $out/bin/stremio \
+      --prefix PATH ":" ${lib.makeBinPath [ ffmpeg ]}
```

```suggestion
      --suffix PATH ":" ${lib.makeBinPath [ ffmpeg ]}
```
we should probably fall back to that if there is no other ffmpeg in PATH

---

Thanks for the review! I applied the suggested changes.